horace_core/utilities/@sqw/symmetrise_sqw.m

```diff
@@ -96,12 +95,15 @@
 
 %convert the vectors specifying the reflection plane from rlu to the
 %orthonormal frame of the pix array:
+% do not rely on the shift of the image to define symetry plain.
 %vec1=uconv\(v1'-header.uoffset(1:3));
 %vec2=uconv\(v2'-header.uoffset(1:3));
+% the symetry plain should be defined in real hkl, not shifted hkl the
+% image may be expressed in.
```

Comment:
It is unclear how the commented code relates to the warning comments. Could we have a clear simple comments with advice to future coders, without the commented calls.

plain-->plane (FTR while it is fairly obvious here, this typo does cause confusion in other settings)

Reply:
I've added TODO comment about it.  plane fixed